composer.json

```diff
@@ -26,14 +26,14 @@
         "nelmio/security-bundle": "~1.4",
         "doctrine/doctrine-migrations-bundle": "~1.0.0",
         "doctrine/migrations": "~1.0.0|1.0.0@alpha",
-        "mopa/bootstrap-bundle": "3.0.0-RC2",
+        "mopa/bootstrap-bundle": "3.0.0",
```

Comment:
It's part of https://www.pivotaltracker.com/projects/1163646/stories/109795400 to make SSA SF 3 compatible. But seeing as I need to test for that story I'll pick this up then and revert it for now.